core/commands/repo.go

```diff
@@ -95,3 +95,59 @@ order to reclaim hard disk space.
 		},
 	},
 }
+
+var repoStatCmd = &cmds.Command{
+	Helptext: cmds.HelpText{
+		Tagline: "Get stats for the currently used repo.",
+		ShortDescription: `
+'ipfs repo stat' is a plumbing command that will scan the local
+set of stored objects and print repo statistics. It outputs to stdout:
+NumObjects      int number of objects in the local repo
+RepoSize        int size in bytes that the repo is currently taking
+RepoPath        string the path to the repo being currently used	
+`,
+	},
+	Run: func(req cmds.Request, res cmds.Response) {
+		n, err := req.InvocContext().GetNode()
+		if err != nil {
+			res.SetError(err, cmds.ErrNormal)
+			return
+		}
+
+		stat, err := corerepo.RepoStat(n, req.Context())
+		if err != nil {
+			res.SetError(err, cmds.ErrNormal)
+			return
+		}
+
+		res.SetOutput(stat)
+	},
+	Options: []cmds.Option{
+		cmds.BoolOption("human", "Output RepoSize in MiB."),
+	},
+	Type: corerepo.Stat{},
+	Marshalers: cmds.MarshalerMap{
+		cmds.Text: func(res cmds.Response) (io.Reader, error) {
+			stat, ok := res.Output().(*corerepo.Stat)
+			if !ok {
+				return nil, u.ErrCast()
+			}
+
+			human, _, err := res.Request().Option("human").Bool()
+			if err != nil {
+				return nil, err
+			}
+
+			buf := new(bytes.Buffer)
+			fmt.Fprintf(buf, "NumObjects \t %d\n", stat.NumObjects)
+			if human {
+				fmt.Fprintf(buf, "RepoSize (MiB) \t %d\n", stat.RepoSize/1024)
```

Comment:
thanks for catching that @lgierth :) 

check #c10e329, I've added a clause to print it in bytes if it is less than one MiB